### PR TITLE
feat(parsers): Update to-style-dictionary parser - DEV-2828

### PR DESCRIPTION
**Pull request by ChucKN0risK**
Add `fontWeight` in font assets exported by `to-style-dictionary` parser.

### CODE REVIEW

File: parsers/to-style-dictionary/to-style-dictionary.spec.ts [hunk 1]
```diff
@@ -645,9 +650,10 @@ describe('To Style Dictionary', () => {
         });
     });
     it('Font', () => {
-      seeds()
-        .tokens.filter(({ type }) => type === 'font')
+      (seeds()
+        .tokens.filter(({ type }) => type === 'font') as Array<FontToken>)
         .map(font => {
+          console.log('FONT', font)
```
Comment (peschee) [inline parsers/to-style-dictionary/to-style-dictionary.spec.ts:656]: This is probably still a left-over

**Review by AllanMichay — COMMENTED** [review]
Other than the comment by @peschee, LGTM

File: parsers/to-style-dictionary/to-style-dictionary.spec.ts (hunk 1 above)
Comment (ChucKN0risK) [reply parsers/to-style-dictionary/to-style-dictionary.spec.ts]: Thanks! It's fixed.

**Review by AllanMichay — APPROVED** [review]
👏 LGTM

**Pull request merged**